lib/media.js

```diff
@@ -225,10 +227,22 @@ function getStyleFiles(state) {
       });
   }
 
+  // add the layout css
+  if (layoutName) {
+    layoutBasePath = path.join(assetDir, 'css', `${layoutName}.${siteStyleguide}.css`);
```

Comment:
Are (component) variations going to be supported for layouts?

Reply:
not if i can help it!!!!! (in all seriousness, probably not because there would be no logical place to edit the variation in kiln)